### ISSUE TITLE
setting the reference

### ISSUE THREAD

**Issue by fritzsedlazeck** [dantaki/SV2#28]
Hi,
I am trying to use SV2 and I am missing something. 
i have used pip install to download and install, then svs configure and 

sv2 -hg38 with the path to the fasta file that I used. 
when I then run svs2:

sv2 -i reads.RG.bam -v mysvs.vcf -snv freebayes.vcf -p my.ped -O output.vcf

I get after a while the following:
************************************************
 PREPROCESSING COMPLETE <time elapsed: 0:05:17>
************************************************
FATAL ERROR: None fasta for reference hg19 not found. Please configure SV2
[ $ sv2 -hg19 FASTA -hg38 FASTA -mm10 FASTA ] github.com/dantaki/SV2/wiki/Installation#configure
FATAL ERROR: None fasta for reference hg19 not found. Please configure SV2
[ $ sv2 -hg19 FASTA -hg38 FASTA -mm10 FASTA ] github.com/dantaki/SV2/wiki/Installation#configure

As I wrote I did these steps for hg38.. Any suggestions?
Thanks
Fritz

**Comment by dantaki** [comment]
Hey Fritz,

Thanks for the heads up. I actually replicated the bug and realized the problem.

SV2 is showing its age, evident with the dependency on python2.7, and setting the default genome assembly to be hg19. 

Because sv2's default genome argument is `hg19` if you plan to use hg38, **you'll need to pass the `-g hg38` argument to the command**

```
sv2 -i reads.RG.bam -v mysvs.vcf -snv freebayes.vcf -p my.ped -O output.vcf -g hg38
```
Should work now, it's been a while since I ran sv2 and I forgot about the `-g` option. Thanks again for the heads up, and thanks especially for including sv2 in your recent comparison of SV genotypers. And congrats on your recent position! 

Best,

Danny
